A/ADIOS2/build_tarballs.jl

```diff
@@ -75,10 +84,18 @@ products = [
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
+    Dependency(PackageSpec(name="Blosc_jll")),
+    Dependency(PackageSpec(name="Bzip2_jll")),
```

Comment:
Unfortunately, bzip2 is a bit problematic: we have versions 1.0.6 and 1.0.8, but they are ABI-incompatible (the soname for the macOS library is different between the two versions), so you have to choose to use either one or the other.  I suggest using 1.0.8 because here the soname is fixed in a more future-proof way (see #2981), the only downside is that 1.0.8 requires julia v1.6.  Is that a problem?

Reply:
I'll leave out Bzip2 instead.